api/src/main/java/net/kyori/adventure/text/format/TextColor.java

```diff
@@ -261,6 +261,8 @@ public interface TextColor extends Comparable<TextColor>, Examinable, RGBLike, S
    * @param b the upper bound ({@code t=1.0})
    * @return the interpolated value, a color between the two input colors {@code a} and {@code b}
    * @since 4.8.0
+   *
+   * @deprecated see {@link net.kyori.adventure.text.format.gradient.RGBGradient#interpolate(float, TextColor, TextColor)}
```

Comment:
Idk if it's just me, but it seems weird to have interpolation methods on the gradient? I don't really have an better API design but something about that feels off to me. Like, they are just util methods and can be used for other stuff, its a bit weird to tie them to the concept of a gradient?
Will take another look when am more awake tho

Reply:
Maybe it's poor naming choice. But it doesn't make sense to have an interpolation method on TextColor when the exact same method exists in a more comprehensive API

Reply:
The method should stay here on `TextColor`.